src/UI/Components/ServiceInstanceForm/ServiceInstanceForm.tsx

```diff
@@ -263,3 +263,12 @@ export const ServiceInstanceForm: React.FC<Props> = ({
 const StyledForm = styled(Form)`
   min-height: 0;
 `;
+
+const StyledButton = styled(Button)`
```

Comment:
I'm not such a fan of this solution.
I did a quick check and seems like the actionGroup is the culprit here. Replacing the component with the ActionList, and adding the `aria-disabled={isSubmitDisabled || !isEditorValid}` resolves the issue without requiring any css overrides.